ci3.8.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - xarray=2022.10.0
   - xarray-beam=0.3.1
   - ecmwf-api-client=1.6.3
-  - flake8=5.0.4
+  - black=22.12.0
```

Comment:
I was a bit unclear in my instructions. Instead of installing black, we can just install `pyink` (not sure the latest version.

Reply:
In here we can see the latest pyink version as 22.12.0 [pyink releases](https://github.com/google/pyink/releases) 

I understood that we can install packages from github releases though they are not as a pypi package. I tried this.
```
pip install git+https://github.com/google/pyink.git@22.12.0
```
But when I tried to run to test,
```
pyink /workspace/weather-tools/weather_dl/download_pipeline/clients.py
```
it gaves me this error. 

```
Traceback (most recent call last):
  File "/workspace/.pyenv_mirror/user/3.8.16/bin/pyink", line 5, in <module>
    from pyink import patched_main
  File "/workspace/.pyenv_mirror/user/current/lib/python3.8/site-packages/pyink/__init__.py", line 814, in <module>
    lines: Optional[Collection[tuple[int, int]]] = None,
TypeError: 'type' object is not subscriptable
```
In here I saw another approach but with vscode [A usage of pyink formatter ](https://github.com/google-research/jax3d/commit/7b81812661aed362e0587ac34fcfaf5cf624f9a0
)

I'm littlebit of unclear what to do next :/

Hope you'll have some idea about this !